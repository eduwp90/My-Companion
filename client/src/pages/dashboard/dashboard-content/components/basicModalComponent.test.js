import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import BasicModalComponent from './basicModalComponent';
import Data from '../../../../helpers/data';
import {
  isOpen, 
  onClose,
} from '@chakra-ui/icons'

const handleChange = e => {
  console.log('handled');
};
const selectIcon = () => {
  console.log('selected')
}

let filledIn = {
  name: 'Dog Name',
  gender: 'Female',
  breed: 'Mix',
  dateOfBirth: '2020-01-01',
  color: 'Brown',
  chipId: '123',
};

let notFilledIn = {
  name: '',
  gender: '',
  breed: '',
  dateOfBirth: '',
  color: '',
  chipId: '',
}

let treatment = [0, {treatment:"vacination"}]
let reminder = 'some reminder'

describe ('BasicModalComponent', () => {
  test ('BasicModalComponent should match snapshot', () => {
    const { container } = render(<BasicModalComponent isOpen={isOpen} onClose={onClose} handleChange={handleChange} treatment={treatment} reminder={reminder} selectIcon={selectIcon} />);
    expect (container.firstChild).toMatchSnapshot();
  })

})