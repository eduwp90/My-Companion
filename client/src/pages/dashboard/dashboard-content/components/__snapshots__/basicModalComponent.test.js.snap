// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicModalComponent BasicModalComponent should match snapshot 1`] = `<div />`;
